### DISCUSSION TITLE
how to upgrade flawlesly (from git repo), esp. avoid long, painful sync?

### DISCUSSION THREAD

**Discussion by ardabro** [Chia-Network/chia-blockchain#1857] [Q&A] [answered]
git pull origin/latest (with all processes stopped) + install + activate is all  I need?
Do I lose anything during this operation?
Especially I'd like to avoid re-syncing to the blockchain again.

**Comment by hoffmang9** [comment, marked as answer]
That should work and your dbs will start from where you shut down.
